Update README.md (#52)
osdc/TabuRei: README.md

## Installation
- Install via [Firefox Store](https://addons.mozilla.org/en-US/firefox/addon/taburei).

> - To be released for Chrome soon. (You can use the developer mode in the meantime)

Reviewer: ```suggestion
- To be released on the Chrome Store soon. You can follow the [Developer Setup](https://github.com/osdc/TabuRei#chrome) meanwhile to use it.
```
